edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/ISystemComponentIdProvider.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    public interface ISystemComponentIdProvider
+    {
+        string EdgeHubBridgeId { get; }
```

Comment:
Maybe we can make this a const and remove this interface?

Reply:
It has the device id in it, so the content is dynamic. I was considering simple indexed/keyed string parameter, and register the current id with autofac, but then a key attribute needed to add in MqttBrokerAdapter, so I would have needed to add an autofac dependency to the project. That is how I chose to create a separate interface, so I didn't need keyed string for injection